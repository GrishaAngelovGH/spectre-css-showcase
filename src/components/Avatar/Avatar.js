const Avatar = ({ initials }) => (
  <figure className='avatar avatar-sm mx-2' data-initial={initials} style={{ background: '#5755d9' }}></figure>
)

export default Avatar