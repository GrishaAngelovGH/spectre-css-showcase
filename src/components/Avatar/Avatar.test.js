import { render } from '@testing-library/react'

import Avatar from './Avatar'

test('should render Avatar component', () => {
  const view = render(<Avatar initials='JS' />)

  expect(view).toMatchSnapshot()
})