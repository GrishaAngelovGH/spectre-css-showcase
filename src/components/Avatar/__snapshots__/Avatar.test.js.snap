// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Avatar component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <figure
        class="avatar avatar-sm mx-2"
        data-initial="JS"
        style="background: rgb(87, 85, 217);"
      />
    </div>
  </body>,
  "container": <div>
    <figure
      class="avatar avatar-sm mx-2"
      data-initial="JS"
      style="background: rgb(87, 85, 217);"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
